README.lensing

```diff
@@ -0,0 +1,22 @@
+To generate the FITS files of the lensed hosts, you will need to run the followi
+ng commands only once:
+
+python lensed_hosts/generate_lensed_hosts_agn.py
```

Comment:
It looks like you have to be in `lensed_hosts` to run this code. So this should be changed. Should also note that the ell_lef.dat file needs to be in the same data directory as your truth databases. Maybe this should change?

Reply:
You could allow the database files to be specified with a command line argument while hardcoding the ell_lef.dat file perhaps.

Reply:
Hi @jbkalmbach, if you run the command above, you could run the programs just by being in the SLSprinkler directory.  Would that work or are we looking to run the FITS stamp generator programs in some other way?

Reply:
It didn't work outside of the `lensed_hosts` directory because the data directory is hard coded. Does it not cause an error when you do that?